### DISCUSSION TITLE
suggestion for collections in lazer

### DISCUSSION THREAD

**Discussion by ahrl9** [ppy/osu#17228] [Ideas]
I haven't found nay similar ideas, so here are some suggestions about collections that would make them easier and more practically to use from my perspective:

1. **checkboxes** - You could enable or disable the collections you wan't to see. If you enable more than one, the maps in the selected ones are visible and all the other maps are hidden.

2. **custom order** - currently people are adding weird letters/numbers/symbols infront of their collections to make them sort properly, it would make the collectins wy more polished and easier to use if you could just drag them in the right order.

3. (this one is not necessary but would be cool) **onscreen collection add/remove button** - add buttons to the beatmap details to add/remove the map/mapset to a collection.

I hope i haven't made a duplicate discussion, if so than i'm sorry :)

**Comment by smoogipoo** [comment]
@arflyte This is more of a memo for you to consider the UI around collections in the new song select designs. It's very hacked in right now.
